db.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const Sequelize = require('sequelize');
+
+// Create the url based on the config file .env in the root directory
+const DB_URL = `mysql://${process.env.DB_USER}:${process.env.DB_PASSWORD}@${process.env.HOST}:${process.env.DB_PORT}/${process.env.DB_NAME}`;
+
+// Connect to database
+const dbConnection = new Sequelize(DB_URL, {
+  logging: false
+});
+
+dbConnection
+  .authenticate()
+  .then(() => {
+    console.log(`✅ Connection to database ${process.env.DB_NAME} at ${process.env.HOST}:${process.env.DB_PORT} has been established successfully.`);
```

Comment:
Coool usage of emojis, you can try [Chalk](https://www.npmjs.com/package/chalk) for your fancy log outputs